tap_zendesk_chat/schemas/agents.json

```diff
@@ -103,6 +103,13 @@
       ],
       "format": "date-time"
     }
```

Comment:
On this line, there should be a comma to separate `last_login` from `scope`. I didn't catch this sooner because I thought the highlighting on `:` was related to the `]`